python/cuda_cccl/tests/parallel/test_unique_by_key.py

```diff
@@ -163,7 +163,19 @@ def test_unique_by_key(dtype, num_items):
     "dtype, num_items",
     DTYPE_SIZE_PAIRS,
 )
-def test_unique_by_key_iterators(dtype, num_items):
+def test_unique_by_key_iterators(dtype, num_items, monkeypatch):
+    cc_major, _ = numba.cuda.get_current_device().compute_capability
+    # Skip sass verification for CC 9.0+, due to a bug in NVRTC.
+    # TODO: add NVRTC version check, ref nvbug 5243118
+    if cc_major >= 9:
+        import cuda.cccl.parallel.experimental._cccl_interop
+
+        monkeypatch.setattr(
+            cuda.cccl.parallel.experimental._cccl_interop,
+            "_check_sass",
+            False,
+        )
+
```

Comment:
Just a note to self that if we find another instance of needing to skip the SASS check, we should look for a DRY solution. I'm OK with one repetition.